docs/users/jsc.rst

```diff
@@ -3,64 +3,128 @@
 Jülich Supercomputing Centre (JSC) - Germany
 ============================================
 
-:website: http://www.fz-juelich.de/ias/jsc
+:website: http://www.fz-juelich.de/jsc
 
-:institute: Forschungszentrum Jülich - http://www.fz-juelich.de/portal/EN
+:institution: Forschungszentrum Jülich - http://www.fz-juelich.de/portal/EN
 
-:contact: *(to be completed)*
+:contact: Damian Alvarez (d.alvarez@fz-juelich.de)
 
-*(last update: Oct 23rd 2016)*
+*(last update: Feb 24th 2017)*
 
 .. _users_use_cases_jsc_history:
 
 History
 -------
 
-*(to be completed)*
+The first contact between JSC and EasyBuild was through the LinkSCEEM (TODO: add url) project in 2013. During that period the
+main x86_64 cluster at JSC -JUROPA- was approaching its end of life. Before its decomission, a test system was deployed. 
+That test system was the precursor for JUROPA's successor, named JURECA. Thus, it was the right time to evaluate 
+EasyBuild as an alternative to have consistent and efficient software deployment. The evaluation period concluded and it
+was decided that EasyBuild would be JSC's tool of choice to manage software in JURECA.
 
 .. _users_use_cases_jsc_systems:
 
 Systems
 -------
 
-*(to be completed)*
+At the moment, EasyBuild is used in JSC'S main production system and a 2 internal systems
+
+JURECA
+~~~~~~
+* Operating system: CentOS Linux 7
+* Processor architecture: Intel Haswell
+* Number of nodes: 1872
+* GPUs: NVIDIA K80 and K20x
+* Interconnects: InfiniBand
```

Comment:
include link to http://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JURECA/JURECA_node.html?